src/System.Private.CoreLib/shared/System/Diagnostics/StackTrace.cs

```diff
@@ -25,7 +26,7 @@ public partial class StackTrace
         /// <summary>
         /// Stack frames comprising this stack trace.
         /// </summary>
-        private StackFrame[] _stackFrames;
+        private StackFrame?[]? _stackFrames;
```

Comment:
I know _stackFrames itself can be null.  But in what situation could a non-null StackFrame[] contain a null?

Reply:
Ah, I see:
```C#
new StackTrace((StackFrame?)null).GetFrames()[0] == null
```